### PR TITLE
 Add Metrics Web App

### PR DESCRIPTION
**Pull request by Jawad-Nazari**
### In this PR, I did the following:

- Create react app using CRA (create-react-app)
- Use GitFlow to introduce new features 💡
- Setup linters 
- Add navigation
- Add home page
- Used React documentation, react components, react props, React Router.
- Connected React and Redux and handled events in a React app.
- Used React hooks.
- Used store, actions, and reducers in React.
- Wrote integration tests with a React testing library.
- Used styles in a React app.
- Applied React best practices and language style guides in code.
- Cleared All Linter errors
- Updated the Read Me file
- Deployed my project on Render

### CODE REVIEW

File: src/test/CryptoDashboard.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Provider } from 'react-redux';
```
Comment (gemmen29) [inline src/test/CryptoDashboard.test.js:1]: - [ ] Kindly add unit tests for pure functions in Redux code as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests%20for%20pure%20functions%20in%20your%20Redux%20code.).
- [ ] Good job on implementing the snapshot tests for components, but Kindly add integration tests using React Testing Library as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20integration%20tests%20for%20your%20application%20using%20the%20React%20Testing%20Library).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Jawad-Nazari, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/CryptoDashboard.test.js (hunk 1 above)
Comment (Jawad-Nazari) [reply src/test/CryptoDashboard.test.js:1]: Hi @gemmen29 ,
Thanks for your time and for reviewing my code. I will apply the necessary changes as you advised. 🙏

Comment (peterrobert) [reply src/test/CryptoDashboard.test.js:1]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Jawad-Nazari, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️
[ ] Great work on this capstone project. The design looks great ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**